xonsh/ptk2/__init__.py

```diff
@@ -0,0 +1,11 @@
+# provide backward compatibility for external xontribs till they can catch up.
+import xonsh.ptk_shell.completer as completer
+import xonsh.ptk_shell.history as history
+import xonsh.ptk_shell.key_bindings as key_bindings
+import xonsh.ptk_shell.shell as shell
+import xonsh.ptk_shell as par
+
+import sys
+
+for m in [par, completer, history, key_bindings, shell]:
+    sys.modules[m.__name__.replace("ptk_shell", "ptk2")] = m
```

Comment:
I think it is cleaner to just have the respective files here which do `from xonsh.ptk_shell.x import *`

Reply:
That works.  Though it does not import global names that start with '_', so could break external developers who weren't following the rules.